src/sagemaker/fw_utils.py

```diff
@@ -257,6 +258,9 @@ def create_image_uri(
     if py_version and py_version not in VALID_PY_VERSIONS:
         raise ValueError("invalid py_version argument: {}".format(py_version))
 
+    if py_version == "py37" and framework not in PY37_SUPPORTED_FRAMEWORKS:
+        raise ValueError("{} does not support Python 3.7 yet.".format(framework))
```

Comment:
maybe s/yet/at this time. There might some some frameworks that never receive Python 3.7 support.

Reply:
Done.